cinema/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255)
+    description = serializers.CharField(max_length=255, required=False)
+    duration = serializers.IntegerField(required=True)
```

Comment:
`reuired=True` is a default value 
https://www.django-rest-framework.org/api-guide/fields/#required